server/runtime/missing_tool_output_repair.go

```
@@ -0,0 +1,760 @@
+			}
+			s.invalidOutputs[callID][kind]++
+			return
+		}
+		if s.materializedOutputs[callID] == nil {
+			s.materializedOutputs[callID] = make(map[repairToolCallKind]int)
+		}
+		s.materializedOutputs[callID][kind]++
+	}
+}
+
+func (s *missingToolOutputRepairScan) unfinishedCalls() map[string]struct{} {
```

golangci-lint (2.12.2)

[error] func (*missingToolOutputRepairScan).unfinishedCalls is unused

(unused)